CMakeLists.txt

```diff
@@ -92,6 +114,23 @@ if(WITH_MMAL)
 	list(APPEND SRC_FILES mmalcam.c)
 endif(WITH_MMAL)
 
+message("-- Configuration: " )
```

Comment:
This shouldn't be needed, they are not interesting for every run and you can list them with `cmake . -LA` for debugging.